code/modules/modular_computers/file_system/programs/command/comm.dm

```diff
@@ -305,10 +311,10 @@ var/last_message_id = 0
 	for (var/datum/comm_message_listener/l in comm_message_listeners)
 		l.Add(message)
 
-	for (var/obj/item/modular_computer/computer in get_listeners_by_type("modular_computers", /obj/item/modular_computer))
-		if(computer && computer.working && !!computer.nano_printer)
+	for (var/obj/item/modular_computer/computer in get_listeners_by_type(LISTENER_MODULAR_COMPUTER, /obj/item/modular_computer))
+		if(computer?.working && !!computer.nano_printer && computer.hard_drive?.stored_files.len)
```

Comment:
What is the purpose of the added check here?

Reply:
I tested the code locally on Runtime, and it runtimed at that exact spot. The check stops the runtime from happening